### PR TITLE
DDF-3911 Updated jacoco in all pom files to be more lenient

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
- Upgraded jacoco and ddf-support versions.
- Modified all pom files to specify the lenient limit to account for differences in compiler and OS during coverage validation.
- Added suppression for sun certs import to test class.

#### Who is reviewing it? 
@figliold 
@jhunzik 
@tbatie 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams/test

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)
CI build should pass.

#### What are the relevant tickets?
[DDF-3911](https://codice.atlassian.net/browse/DDF-3911)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -246,7 +246,8 @@
         <maven-war-plugin.version>2.2</maven-war-plugin.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
         <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
-
+	<maven-jacoco-plugin.version>0.8.1</maven-jacoco-plugin.version>
```
Comment (Bdthomson) [inline pom.xml]: ✏️ Can you indent this so it's in line with the rest? Also looks like there are new lines above and below it.

File: pom.xml [hunk 2]
```diff
@@ -402,7 +403,14 @@ are essentially embedding a slightly modified version of felix -->
                 <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
-                    <version>0.8.0</version>
+                    <version>${maven-jacoco-plugin.version}</version>
+		    <dependencies>
```
Comment (Bdthomson) [inline pom.xml]: Are you missing a `</plugin>` tag right here?

Comment (paouelle) [reply pom.xml]: No I am not. The plugin tag is below at line 826

File: pom.xml (hunk 1 above)
Comment (paouelle) [reply pom.xml]: 👍 I would say that this is the problem with no official formatters, you end up with difference in files 😞

**Comment by paouelle** [comment]
build now
rebased

**Comment by shaundmorris** [comment]
Do we need to verify a build on Windows? I think the CI build still does Linux only

**Comment by paouelle** [comment]
not sure we need to verify the Windows build. Since at the minimum, we are not doing anything worst than before
in fact we are more lenient

**Pull request merged**